docs/components/Combobox/Web.stories.tsx

```diff
@@ -64,8 +65,9 @@ const ComboboxButton: ComponentStory<typeof Combobox> = args => {
 
 const ComboboxChip: ComponentStory<typeof Combobox> = args => {
   const [selected, setSelected] = useState<ComboboxOption[]>([]);
+
   return (
-    <Combobox {...args}>
+    <Combobox {...args} multiSelect>
```

Comment:
oh I just noticed this. was this intentional to make the chip example a multiselect?

Reply:
Not really - I can't remember why I added it but I was testing something in an earlier iteration. I'll remove this!

Reply:
Done here fda87b77dc2f1061717036b0909a17b0bf0c9c21